examples/complete/main.tf

```diff
@@ -1,28 +1,22 @@
 module "complete_efs_example" {
-  source                              = "boldlink/efs/aws"
-  creation_token                      = local.name
+  source                              = "../../"
+  creation_token                      = var.name
   efs_mount_target_subnet_ids         = local.public_subnets
   efs_file_system_policy              = local.efs_file_system_policy
   vpc_id                              = local.vpc_id
   transition_to_ia                    = ["AFTER_90_DAYS"]
   transition_to_primary_storage_class = ["AFTER_1_ACCESS"]
-  ingress_rules = {
-    efs_ingress = {
-      cidr_blocks = [local.vpc_cidr]
-      description = "efs ingress rule"
+  tags                                = var.tags
+  external_security_groups            = [data.aws_security_group.default.id]
+  create_security_group               = false
```

Comment:
wouldn't the "complete" example create its own security group as well? If no, I think we should remove the "security_group_ingress" from here because it won't be used. Correct me if I'm missing something, please ;)

Reply:
Hey @mariomerco The example creates its own security group. The intention of this example (from the fix) is to use more than 1 security group; 1 created by the module and one external security group so there can be a total of 2 security groups.

The security group ingress is for the security group created by the module